packages/main/src/ToolbarButton.hbs

```diff
@@ -0,0 +1,15 @@
+<ui5-button
+	class="ui5-tb-button ui5-tb-item {{this.classList.value}}"
```

Comment:
Hi @vladitasev, @ilhan007,

Do we support this kind of propagation css classes? I'm not sure but instead of doing this magic with propagating the css classes and styles applied to the abstract elements we can I think that CSS part would fit here.